docs/source/monitoring/overview.md

```diff
@@ -0,0 +1,3 @@
+# Monitoring Overview
+
+Jupyterhub operational metrics are collected using Prometheus. Please refer to [this doc](https://github.com/jupyterhub/mybinder.org-deploy/blob/master/docs/source/components/metrics.md) for a primer on Prometheus.
```

Comment:
Maybe there's a better link to a prometheus primer, e.g. the prometheus docs themselves?